### ISSUE TITLE
FYI Gramps project has an addon for this package.

### ISSUE THREAD

**Issue by daleathan** [T-F-S/genealogytree#16]
Hi @T-F-S ,

Great charts, really appreciated that you created them 👍 

You might like to know that the Genealogy prgram Gramps now has a third party addon that creates charts using your CTAN package (not sure how).

According to the author it can manually be added to the current Gramps version, and is available via the built in addons  for the yet to be released Gramps 5.

-
Dale

See Feature request:
10223: LaTeX genealogytree reports
https://gramps-project.org/bugs/view.php?id=10223

Source code:
Add four new genealogy tree reports 
https://github.com/gramps-project/addons-source/commit/921dd7c1fb302d20b7dd9c4dbc3316a34b2fb227

**Comment by T-F-S** [comment]
Thank you for that exciting news. I was not aware of this addon, but it is great that somebody (Nick Hall?) created one.

Until now, i did not use Gramps before, but now I think I have to ... ;-)

`genealogtree` was designed to work with generated content from a genealogy software, but there was no public software available until now. So, this is really good news.

I have not tried this addon yet, but I will do in the near future.

**Comment by mikkelee** [comment]
By the way, I have written a perl script that I use to generate trees. I'll clean it up and put it here on github, maybe others will find a use for it.

**Comment by T-F-S** [comment]
Good idea. Please, add a link to this perl script as a further comment when you have it on github.

**Comment by mikkelee** [comment]
Here we go. It's very simple! :)

https://github.com/mikkelee/genealogytree-mkgraph

**Comment by jojomi** [comment]
I did something similar to @mikkelee in [Go](https://golang.org), mainly to scratch my own itch actually and process data I already had outside of Gedcom files. I created a yaml based file format that suited my needs and a bit of helper code for making nice documents.

Here's the link and some pictures: https://github.com/jojomi/generations

I would suggest to collect those tools and scripts on a [wiki page](https://github.com/T-F-S/genealogytree/wiki), what do you think @T-F-S?

p.s. thank you for creating (and documenting!) this awesome software and even making it available for all of us!

**Comment by mikkelee** [comment]
I've realized some shortcomings in my script:

A: every family is a `union` even though most don't have to be
 -- (@T-F-S persons without a co-parent but who are nevertheless assigned a `union`, their family edge is off to the side even though there's nothing to the "other side". Dropping down a level works).

B: it took me over an hour to figure out how to change the box thickness. this worked but idk if its proper `{box={boxrule=0.75mm}}`

C: also it'd be good to mention that the node text by default uses `sffamily`(?)

**Comment by T-F-S** [comment]
@jojomi To collect third-party tools and scripts here is not in the scope of the package and could impose several problems of different nature...

But, it may be a good idea to have a wiki page with LINKS to tools and scripts like your own.

**Comment by T-F-S** [comment]
@mikkelee To your questions:

A: A `union` always means that this is the second, third, etc. relation of the `g` node. So, even if no further persons are added to the standard (first) relation, some space is reserved. Maybe, your script could be adapted to count the relations first and do some distinction of cases (?).

B: Yes, that is correct. The `box` option is the right place to change box parameters like box thickness, color, font, etc. Depending on the selected `node processor`, these settings are typically `tcolorbox` options.

C: No, the default setting of a node does not use `\sffamily`. The default setting is the leave the document font unchanged. Many, but not all, templates from the `template` library do set the font to `\sffamily`. Also, templates set a lot of other options for the desired output.

By the way: The source code of the `templates` library may be helpful to create own designs. Generally, it is a collection of ready-to-use examples.

**Comment by jojomi** [comment]
@T-F-S That was what I had thought about, collecting links and short descriptions in a list that is easier to follow than this thread. So that's what I did! https://github.com/T-F-S/genealogytree/wiki/Auxiliary-Tools
Feeback and more info is welcome @mikkelee  @daleathan :).

**Comment by T-F-S** [comment]
@jojomi Thank you - that list looks good :-)

**Comment by mikkelee** [comment]
@T-F-S Thanks for the answers, I'll update my script to only use `union` when necessary at some point soon.

**Comment by torfsen** [comment]
Thanks for the wiki page, @jojomi. I've just added [`gedcom2gtr`](https://github.com/torfsen/gedcom2gtr). However, I'm not sure how discoverable the page is for the average user. Perhaps @T-F-S can include a link to the wiki page in the README.

**Comment by T-F-S** [comment]
Good idea. I just added a link in the README.
